### PR TITLE
Add news post for release-20181215.

### PR DESCRIPTION
**Pull request by pchote**
TODO:
 - [x] Finish placeholder text (CC @icereaper)
 - [x] Add battlefield news

### CODE REVIEW

File: content/news/release-20181215-kknd.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+While work continues on the next OpenRA playtest, we have collected a number of bugfixes that we wanted to fix before the holiday season. We are today pushing a bugfix release, [release-20181215](/download/), with the following improvements:
+
+* Fixed servers showing as "Waiting" in the server list after they have started.
+* Fixed a crash in the lobby if the admin changes map while a player is joining.
+* Fixed part of the lobby UI disappearing after transferring admin to another player.
+* Fixed players as "Anonymous" in the lobby after switching from another mod
```
Comment (reaperrr) [inline content/news/release-20181215-kknd.md]: 'showing as'

File: content/news/release-20181215-kknd.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+* Fixed a crash when using the /levelup cheat with units that are already veteran.
+* Fixed rare crashes when using Attack Move, and when ordering aircraft to land.
+* Fixed bugs with the Chronoshift killing Construction Yards and duplicating MCVs.
+* Fixed with tech-structures if the owner surrenders while it is being captured.
```
Comment (reaperrr) [inline content/news/release-20181215-kknd.md]: 'Fixed bug with'

File: content/news/release-20181215-kknd.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+* Fixed a rare bug where units become uncontrollable after exiting a transport.
+* Fixed compatibilty issues with certain older GPUs.
+* Removed tech-structure abilities from campaign-specific buildings in Red Alert.
+
+Head over to the [downloads page](/download/) to grab the new release, and spread the word!
+
+<hr />
+
+In other news, we are pleased to announce the first public alpha of the [Krush Kill 'n' Destroy remake](https://www.kknd-game.com/) project, which aims to rebuild and improve on the first two KKnD games much like OpenRA has done for the classic C&C games.
```
Comment (reaperrr) [inline content/news/release-20181215-kknd.md]: Although most readers probably know or can guess, maybe change the last part before the comma to "(...) project based on OpenRA's engine, (...)" to make clear it's not some unrelated project.

File: content/news/release-20181215-kknd.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+* Fixed servers showing as "Waiting" in the server list after they have started.
+* Fixed a crash in the lobby if the admin changes map while a player is joining.
+* Fixed part of the lobby UI disappearing after transferring admin to another player.
+* Fixed players as "Anonymous" in the lobby after switching from another mod
+* Fixed a crash when using the /levelup cheat with units that are already veteran.
+* Fixed rare crashes when using Attack Move, and when ordering aircraft to land.
+* Fixed bugs with the Chronoshift killing Construction Yards and duplicating MCVs.
+* Fixed with tech-structures if the owner surrenders while it is being captured.
+* Fixed a rare bug where units become uncontrollable after exiting a transport.
+* Fixed compatibilty issues with certain older GPUs.
```
Comment (reaperrr) [inline content/news/release-20181215-kknd.md]: compatibil*i*ty

File: content/news/release-20181215-kknd.md (hunk 2 above)
Comment (obrakmann) [reply content/news/release-20181215-kknd.md]: also s/it is/they are/

**Comment by pchote** [comment]
Updated.

File: content/news/release-20181215-kknd.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+While work continues on the next OpenRA playtest, we have collected a number of bugfixes that we wanted to fix before the holiday season. We are today pushing a bugfix release, [release-20181215](/download/), with the following improvements:
+
+* Fixed servers showing as "Waiting" in the server list after they have started.
+* Fixed a crash in the lobby if the admin changes map while a player is joining.
+* Fixed part of the lobby UI disappearing after transferring admin to another player.
+* Fixed players showing as "Anonymous" in the lobby after switching from another mod
+* Fixed a crash when using the /levelup cheat with units that are already veteran.
+* Fixed rare crashes when using Attack Move, and when ordering aircraft to land.
+* Fixed bugs with the Chronoshift killing Construction Yards and duplicating MCVs.
+* Fixed a glitch if the owner surrenders while a tech-structures is being captured.
```
Comment (obrakmann) [inline content/news/release-20181215-kknd.md]: a tech-structure, no plural.

File: content/news/release-20181215-kknd.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+* Fixed compatibility issues with certain older GPUs.
+* Removed tech-structure abilities from campaign-specific buildings in Red Alert.
+
+Head over to the [downloads page](/download/) to grab the new release, and spread the word!
+
+<hr />
+
+In other news, we are pleased to announce the first public alpha of the [Krush Kill 'n' Destroy remake](https://www.kknd-game.com/) project, which aims to rebuild and improve the first two KKnD games using the OpenRA game engine.
+
+[blurb from IceReaperrr: a couple of sentences about what KKnD is and the history/plan of/for the project]
```
Comment (IceReaper) [inline content/news/release-20181215-kknd.md]: After the destruction of earths surface by the great nuclear war, those unlucky to survive mutated due to the high radiation. To make sure this never happens again, the so called evolved found their way back to their spiritual roots and relying on animals. Till 60 years later, the survivors which were hiding underground came back to the surface to claim it causing a war for the overworld.
The KKnD remake project started back in 2011 and is now fully open source along with its first public release. While still in early alpha stage the current release is focused around KKnD 1 and multiplayer gameplay. Further plans for the future include singleplayer gameplay as well as the sequel KKnD 2: Krossfire.
KKnD features unique mechanics such as an individual upgrade system and a parallel contruction system. Besides that it comes with a lot of small tweaks and enhancements over the original game to meet up modern standards.

**Comment by pchote** [comment]
Updated:

screenshot 2018-12-16 at 17 04 50

File: content/gamenews [hunk 7]
```diff
@@ -1,4 +1,4 @@
...
+2018-12-16 09-00-00:
+	Title: 20181215 bugfix release
+	Author: Paul Chote
+	Content: A new release is now available with fixes for a collection of frustrating bugs and crashes in release-20180923:\n\n • Started servers showing as "Waiting" in the server list\n • Crashes and visual glitches in the game lobby\n • Rare crashes with attack-move and aircraft land orders\n • Chronoshift-MCV death and duplication glitches\n • Crashes with the /levelup cheat\n • Transport unload glitch\n • Tech-structure surrender-capture glitch\n • Compatibility issues certain older GPUs\n\nVisit https://www.openra.net to download the new release and to find out about the public KKnD alpha!
```
Comment (reaperrr) [inline content/gamenews]: Compatibility issues *with* certain older GPUs

**Comment by reaperrr** [comment]
👍 after that.

**Comment by pchote** [comment]
Fixed.

**Pull request merged**